plugin/trino-elasticsearch/src/main/java/io/trino/plugin/elasticsearch/BuiltinColumns.java

```diff
@@ -45,6 +45,16 @@
         this.supportsPredicates = supportsPredicates;
     }
 
+    public static Optional<BuiltinColumns> of(String name)
```

Comment:
`forName` (or at least `for`); `of` is suitable when creating value-based object and providing the value for it, and not suitable for lookup-kind operations